install/traf_checkset_env.sh

```diff
@@ -244,11 +244,14 @@ if [ ! -d ${javapath} ]; then
      exit 1
 fi
 
+source $HOME/.bashrc
 javahome=`grep JAVA_HOME ~/.bashrc | wc -l`
 if [ "x${javahome}" = "x0" ]; then
     echo -en "\n# Added by traf_checkset_env.sh of trafodion\n" >> $HOME/.bashrc
     echo -en "export JAVA_HOME=${javapath}\n" >> $HOME/.bashrc
     echo -en "export PATH=\$PATH:\$JAVA_HOME/bin\n" >> $HOME/.bashrc
```

Comment:
See comment above. Writing into the user's .bashrc is probably not a very good idea. You can create a file ~/.trafodion instead.